services/brig/test/integration/API/OAuth.hs

```diff
@@ -157,11 +158,11 @@ testCreateOAuthCodeSuccess brig = do
 testCreateOAuthCodeRedirectUrlMismatch :: Brig -> Http ()
 testCreateOAuthCodeRedirectUrlMismatch brig = do
   let newOAuthClient = newOAuthClientRequestBody "E Corp" "https://example.com"
-  cid <- occClientId <$> registerNewOAuthClient brig newOAuthClient
+  c <- registerNewOAuthClient brig newOAuthClient
```

Comment:
oh no, no single-letter identifiers please!  :)

anyway this is great stuff, i'll start using this everywhere now!